models/staging/edfi_3/stage/stg_ef3__student_academic_records.sql

```diff
@@ -25,7 +23,7 @@ deduped as (
     {{
         dbt_utils.deduplicate(
             relation='keyed',
-            partition_by='k_student_academic_record, ed_org_id',
+            partition_by='k_student_academic_record',
             order_by='pull_timestamp desc'
```

Comment:
can we add `api_year desc` in the order by to keep the later ODS as source of truth where there's overlap?